code/modules/economy/supply_packs.dm

```diff
@@ -216,6 +216,15 @@ ABSTRACT_TYPE(/datum/supply_packs)
 	containertype = /obj/storage/crate/freezer
 	containername = "Catering: Fresh Produce Crate"
 
+/datum/supply_packs/pizza_boxes
+	name = "Pizza Boxes"
+	desc = "x3 Pizza Boxes"
+	category = "Civilian Department"
+	contains = list(/obj/item/plate/pizza_box = 3)
+	cost = 2000
+	containertype = /obj/storage/crate/wooden
+	containername = "Pizza Boxes inside Box"
```

Comment:
so good a+

Reply:
i changed it, but i'm not sure if it's much better, so i'll leave this review open until you say it's fine.

Reply:
no like i really liked it

Reply:
well, since i got a better idea than using cargo, i can mark this as resolved.